src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -984,8 +956,7 @@
     (let [subject (get-csr-subject csr)]
       (sling/throw+
        {:kind :disallowed-extension
-        :msg (format (str "CSR '%s' contains subject alternative names (%s), which are disallowed. "
-                          "Use `puppet cert --allow-dns-alt-names sign %s` to sign this request.")
+        :msg (tru "CSR ''{0}'' contains subject alternative names ({1}), which are disallowed. Use `puppet cert --allow-dns-alt-names sign {2}` to sign this request."
```

Comment:
Noted. I will change that. I'll write a translator comment so that the translator will understand that it is a command to be used by the user.